assets/app/view/game/round/auction.rb

```diff
@@ -170,7 +170,7 @@ def render_input(company)
               ]
             end
 
-          if @step.may_reduce?(company)
+          if @step.respond_to?(:may_reduce) and @step.may_reduce?(company)
```

Comment:
This appears to work in the compiled JS, but if this was executing in Ruby it wouldn't work; the method is `may_reduce?` so the symbol would need to be `:may_reduce?` rather than `may_reduce`

```suggestion
          if @step.respond_to?(:may_reduce?) and @step.may_reduce?(company)
```